frontend/src/utils/api/course.js

```diff
@@ -0,0 +1,31 @@
+import { get } from './index'
```

Comment:
u can just write as from '.'

Reply:
ok, but this is copied from your code in questions.js

Reply:
I make mistakes too 👍